#Single-cell 
[Theis Lab](https://www.sc-best-practices.org/preamble.html)
## Single-cell best practices
1. Introduction
Single-cell ribonucleic acid ([RNA](https://www.sc-best-practices.org/glossary.html#term-RNA)) sequencing assays

- Cells are the builiding block of life. Cells the basic unit of life, are of different types. They evolve during their development. Their function, shape and gene expression profiles ( the genes that they express) can change due to disease or other causes. If there are strong disruptions in the system  that affect and cause the cells to not be regulated this can affect the whole system and lead to diseases as cancer. 

- Understanding how cells work can be done in many ways, one of them is to study the gene expression profile of each individual cell. Initially only the transcriptome information of the cell was studied. This can be enriched with spatial, chromatic accessbility or protein information.  

- Sequencing: Break down the order of DNA nucleotides.  For a specific DNA segment, for complete genome, of complex micriobiome.  Find location and function of genes, understand open reading frames (ORFs), promoter regions & CpG islands. Applications: Evolutionary analysis, associationd between mutations and diseases among others.  E.g - Sickle  cell disease. Single nucleotide mutation in GAG codon to GTG codon in B-globin gene causing aminoacid substitution. 
2. History of sequencing 
1800 - 1950 - 1960 - 1970-1980 - 1990 - 2000  -2010 - 2020

2.1 First generation sequencing - SANGER 
	- Strengths: Simple and affordable with very low error rate (<0.001%).
	- Limitations: Short pieces of DNA of about 300 to 1000 base pairs (bp). No good quality in the first 15-40 bp ( primer binding zone)
	-   Sanger sequencing is more expensive than second or third generation sequencing per sequenced base.

2.2 Second generation sequencing - next-generation sequencing (NGS)  

-  History
	- 1996, Mostafa Ronaghi, Mathias Uhlen, and Pȧl Nyŕen - pyrosequencing,  Pyrosequencing measures luminescence that is generated by pyrophosphate synthesis during sequencing. “sequencing-by-synthesis”. 
	- 1998 - Shankar Balasubramanian and David Klenerman -  sequencing-by-synthesis process + fluorescent dyes at the company Solexa. 
	- Solexa’s technology also forms the basis of Illumina’s sequencers, which dominate the market today. 
	- 2005 - The Roche 454 sequencer  -  automate the pyrosequencing Many 2007 - SOLiD systems’ “sequencing-by-ligation”
	- 2011 -  Life Technologies’ Ion Torrent - “sequencing-by-synthesis” to detect hydrogen ions when new DNA is synthesized.
	
	Strengths:
	-   Second generation sequencing - cheapest for required chemicals.
	-   High sensitivity to detect low-frequency variants 
	-   Comprehensive genome coverage.
	-   High capacity with sample multiplexing
	-   Sequence thousands of genes simultaneously
	
	Limitations:
	-   The sequencing machines expensive 
	-    Big, stationary machines and not designed for field work.
	-   Results in many short sequencing fragments (reads) which are hard to use for novel genomes
	-   The quality of sequencing result is dependent on the reference genome

2.3 Third generation sequencing - Next generation sequencing 

- Provide longer reads 75 - 300 bp for de novo assembly
- Real time sequencing and portable sequenceres - field work 
- 2010 ZWM (zeero-mode waveguide) sequencing  by PacBio. Nanoholdes single DNA polymerase. Each nucleotide type is labeled with fluorescent dye and the emitted signal wavelenght during incorporation  is read out. Length reads 8-15 kbp 
- 2012 Nanopore produced GridION, MiniIO and Flonge.  Portable sequencers, length up to 2 Mb. Based on electric current changes when nucleic acids go through protein nanopore
	- Strengths:
		- Long reads
		- Portable 
		- Real time sequencing. Faster sequencing. 
		- Allow to decetec epigenetic DNA/RNA modifications directly 
	- Limitations:
		- Expensive 
		- Higher error rate

3. NGS process overview
Sequence DNA = Reverse Transcribe RNA
![[molecular_biology_dogma.png]]

3.1 Steps for NGS:
- Sample and library preparation
Cut DNA samples and ligate them with adapter moleculres. The sequencing libraries are pools of DNA with adapter sequences specific to the sequencing method/device and provide barcodes for sample identifiaction.
- Amplification and sequencing.
Single strand molecules from the library. Through Polimerase Chain Reaction (PCR) copies are created out of the segments and DNA clusters form. Each individual clusters perform a reaction for each run. 
- Data output and analysis: The output depends on the mehtod and also de analysis. 

4. RNA Sequencing. 
- In order to better understand the dynamic and real-time operations of a cell., RNA-Seq is used.  It includes a RNA reverse transcription step and synthetizes cDNA. 
- Sequencing RNA allows to take snapshot of cells, tissues or samples at the precise moment of sequencing. The outcome are gene expression profiles. 
- Use case: How the cells responds to medicine in different conditions, detect gene isoforms, gene fusions, SNP etc. 
- No need of probe design to target region of interst as in RT-qPCR
- 4.1 Overview
	- Sequencing mix of RNA across cells - Bulk sequencing 
		- Cheaper
		- Easier
		- Cell averaged expression profiles
		- Easier to analyze
		- Hide cells expression profile heterogeneity
	
	- Sequencing the transcriptomes individually - Single Cells RNA Seq
		- Experimentally complex
		- Experimentally expensive
		- More difficult downstream analysis
		- Increased resolution
		- Easy to draw false conclusions
		- Gene expression at single cell level
		- USE CASE: Oncology, rare tumour cells resistant to therapy that can cause relapse. 
		- STEPS
			- Lysis of cells 
			- Reverse transcription of RNA
			- Amplification
			- Sequencing
			*EXTRA* (where sc assay differ)
			- Cell isolation
			- Transcript amplification
			- Sequencing
			- Physical separation into smaller reaction chambers/cell labeling
			- Map the transcriptomes back to the cells of origin.
- 4.2 Transcript quantification
Counting the  hits of the sequenced transcripts against the gene sequences.
	- Full lenght TQ: Cover the full transcript with sequencing reads 
		- Only plte-based protocols
		- Similar library preparation as in Bulk-RNA
		- Biased regions in gene body due to uneven coverage of transcripts
		- Detect splice variants
	- Tag based TQ: Only sequence the 5' or 3' ends of the transcripts.
		- Not covering the full gene lenght
		- Difficult to align reads to a transcript 
		- Difficult to differenciate isoforms
		- #UniqueMolecularIdentifiers (UMIs) can be used. 
			- Molecular barcodes / Random barcodes
			- Short random nucleotide sequence added to each molecule. 
			- Added in library generation before amplification
			- Important for downstream analysis. 
			- To solve biases in the transcript amplification process. #AmplificationBiases
			-  To determine original number of molecules in sample is challenging. Quantify the original non-duplicated molecules
			- Allows for normalization of gene counts, without loss of accuracy. 
	IMPORTANT to know, for downstream analysis. 
	
- 4.3 *Transcript amplification*
- PCR (Polymerase chain reaction) creates copies of transcripts
- Early stages PCR (lower error rate) vs Late stages PCR (Higher error rate in amplfied sequences)
- Important step to ensure enough transcripts --> For QC & Sequencing
- #AmplificationBiases 
	- Term to define the RNA/cDNA sequences that tend to be amplfied --> Sequenced more often --> Higher counts 
	- Non-very-active gene can be highly expressed. 
	- Computatinally correction:
		- Remove sequences with identical alignment coordinates
		- Better --> Use UMIs 
			- UMIS allos normalization of gene counts without accuracy loss. 

4.4 Single cell sequengin protocols
Depending on cell isolation protocols:
	- Microfluidic device-based strategies --> Cells into Hydrogels droplets
	![[microfluidic_device_sc.png]]
	- Compartmentalisation into single-cell reaction chambers.
	- The droplet contain beads (perlas) + single cells
	- Microbeads have on-bead primers = PCR handle + cell barcode + 4-8bp UMI + Poly-T tail
		1.  When cell is broken (lysis) --> mRNA released captured by barcoded oligonucleotides from the beads
		2. Collect droplets and break them --> release #single-cell_transcriptomes_attached_to_microparticles ( #STAMPs)
		3. PCR  --> amplify the transcripts
		4. Reverse transcription 
		5. Tagmentation --> Randomly cut transcripts + sequencing adapators attached. 
		RESULT: Sequencing lybrary
	- Only 10% of cell transcripts are recovered. Enough for cell identification.
	- Generate 1000 droplets/second.  
	- High amount of droplets, low cost
	Protocols:
		- inDrop MOST FLEXIBLE, OPEN SOURCE
		- Drop-seq OPEN SOURCE 
		- 10x Genomics Chromium (commercially available) OUTPERFORMS but MORE EXPENSIVE
	- Well plated based --> Physically separated into wells
	![[plate_based.png]]
	- Commercial Fluidigm C1 microfluidic chip --> Loads & separates into small reaction chambers
![[FluidigmC1.png]]
- Nanopore single-cell transcriptome sequencing
From the aboved mentioned methods each has a different application and is best suited for different experimental aims. 
- If budget is limiting --> Choose the most robust and cost effective
- If deep characterization is needed + specific cell type population --> plate-based methods
- If the cell population is heterogeneous --> Droplet based methods

Single-cell vs single nuclei 
- It is important to differenciate when sequencing the information of only the nucleus can be beneficial. 
- There are specific cell type that can be damaged easily and will therefore not be present in the single cell analysis.  
- The availability of fresh cells tissue can also be a problem for specific cell type detection. - 
- However the nucleus information is more resistant to mechanical force and the information can be preserved.





